From 40a699f7a9e54f3a562b50a938e8d9fda5ce516e Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 6 Jun 2017 00:16:31 +0800
Subject: [PATCH 1850/5242] MLK-15008-3 ARM64: dts: freescale: imx8: remove
 local-timer-stop flag

commit  1972ffe3b1e3123be9d47411754af64583c81be3 from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX8QM, as there is no SoC platform broadcast timer
available now, remove the local-timer-stop flag for
cpu-idle driver.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8-ca53.dtsi |    2 --
 arch/arm64/boot/dts/freescale/fsl-imx8-ca72.dtsi |    2 --
 2 files changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8-ca53.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8-ca53.dtsi
index 322c7f5..b1a4e4f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8-ca53.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8-ca53.dtsi
@@ -23,7 +23,6 @@
 
 			CPU_SLEEP: cpu-sleep {
 				compatible = "arm,idle-state";
-				local-timer-stop;
 				arm,psci-suspend-param = <0x0000000>;
 				entry-latency-us = <700>;
 				exit-latency-us = <250>;
@@ -32,7 +31,6 @@
 
 			CLUSTER_SLEEP: cluster-sleep {
 				compatible = "arm,idle-state";
-				local-timer-stop;
 				arm,psci-suspend-param = <0x1000000>;
 				entry-latency-us = <1000>;
 				exit-latency-us = <700>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8-ca72.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8-ca72.dtsi
index 025db05..944c160 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8-ca72.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8-ca72.dtsi
@@ -24,7 +24,6 @@
 
 			CPU_SLEEP: cpu-sleep {
 				compatible = "arm,idle-state";
-				local-timer-stop;
 				arm,psci-suspend-param = <0x0000000>;
 				entry-latency-us = <700>;
 				exit-latency-us = <250>;
@@ -33,7 +32,6 @@
 
 			CLUSTER_SLEEP: cluster-sleep {
 				compatible = "arm,idle-state";
-				local-timer-stop;
 				arm,psci-suspend-param = <0x1000000>;
 				entry-latency-us = <1000>;
 				exit-latency-us = <700>;
-- 
1.7.9.5

